Source/Models/SwishError.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Argo
+
+public enum SwishError {
+  case SessionError(NSError)
+  case ServerError(code: Int, json: AnyObject)
+  case JSONSerializationError(NSError)
+  case ArgoError(DecodeError)
+}
+
+extension SwishError: ErrorType { }
+
+public extension SwishError {
+  var rawError: NSError {
```

Comment:
Yeah, well I suppose the idea of an "underlying" error doesn't really make sense in that case. The response _is_ the error, there's nothing underlying. I suppose you could make the return type optional? But it depends on how this method is intended to be used.